src/Uno.UWP/Storage/StorageFile.Base.cs

```diff
@@ -37,37 +38,42 @@ public void InitOwner(StorageFile owner)
 			public abstract DateTimeOffset DateCreated { get; }
 
 			public bool IsEqual(IStorageItem item)
-				=> item is StorageFile sf && IsEqual(sf._impl);
+				=> item is StorageFile sf && IsEqual(sf.Implementation);
 
-			protected abstract bool IsEqual(ImplementationBase impl);
+			protected abstract bool IsEqual(ImplementationBase implementation);
 
-			public abstract Task<StorageFolder> GetParent(CancellationToken ct);
+			public abstract Task<StorageFolder?> GetParentAsync(CancellationToken ct);
```

Comment:
As we don't have any sync version, I would not have suffix all of them by `Async`

Reply:
I would vote for keeping the suffix if you are not against it strongly, as I then tend to make the mistake of accidentally storing `var parent = GetParent(ct)` for example only to find out it is a `Task` afterward when I can't see the properties I need in IntelliSense 😅 .

Reply:
No it's  a personal preference, I'm fine with it :)